store/src/java/com/zimbra/cs/account/callback/CallbackUtil.java

```diff
@@ -111,4 +114,25 @@ public static boolean isLocalServer (Server server) throws ServiceException {
         Server local = Provisioning.getInstance().getLocalServer();
         return server.getId().equals(local.getId());
     }
+
+    /**
+     * This method is used to validate attribute value
+     * @param attrName
+     * @param attrValue
+     * @param attrs
+     * @throws ServiceException
+     */
+    public static void validateAttributeValue(String attrName, int attrValue, Map<String, Integer> attrs) throws ServiceException {
```

Comment:
This needs renamed to indicate it's specific to validating 2FA attributes and preventing duplicate values, as I assume CallbackUtil is going to get other methods in the future that "validate an attribute value" in some way.